pytensor/tensor/nlinalg.py

```diff
@@ -1031,7 +1032,7 @@ def tensorsolve(a, b, axes=None):
 def kron(a, b):
     """Kronecker product.
 
-    Same as scipy.linalg.kron(a, b).
+    Uses the JAX implementation for kron. 
```

Comment:
We can write `Same as np.kron(a, b)`. 

Actually I'm pretty sure JAX copied numpy and we copied JAX, and hopefully someday someone will copy us.